scripts/README.md

```diff
@@ -97,6 +97,9 @@ please do not add new entries to these files. PRs removing (the need for) entrie
   to the appropriate topic on zulip.
 - `count-trans-deps.py`, `import-graph-report.py` and `import_trans_difference.sh` produce various
   summaries of changes in transitive imports that the `PR_summary` message incorporates.
+- `zulip_emoji_merge_delegate.py` is called every hour by a Github action cronjob.
```

Comment:
This description is now outdate: the number is wrong, and there's a third emoji --- right?